client/components/seo/style.scss

```diff
@@ -50,3 +50,11 @@
 	line-height: 1.4;
 	max-width: 616px;
 }
+
+.meta-title-editor .segmented-control__link {
+	@include breakpoint( "<660px" ) {
```

Comment:
Our preferred method is _mobile-first_, so we should add these attributes instead to the base class and use `breakpoint( ">660px" )` to adjust